clustered/integration-test/src/test/java/org/ehcache/clustered/management/AbstractClusteringManagementTest.java

```diff
@@ -188,6 +189,22 @@ public static void beforeClass() throws Exception {
     sendManagementCallOnEntityToCollectStats();
   }
 
+  protected static void createNmsService() throws ConnectionException, EntityConfigurationException {
+    if (managementConnection != null) {
+      try {
+        managementConnection.close();
+      } catch (IOException ignored) {
+      }
+    }
+    managementConnection = CLUSTER.newConnection();
+
+    NmsEntityFactory entityFactory = new NmsEntityFactory(managementConnection, AbstractClusteringManagementTest.class.getName());
+    NmsEntity tmsAgentEntity = entityFactory.retrieveOrCreate(new NmsConfig());
+
+    nmsService = new DefaultNmsService(tmsAgentEntity);
+    nmsService.setOperationTimeout(5, TimeUnit.SECONDS);
```

Comment:
Is this related to `Timeouts.operationTimeout`? If yes, we should use `Timeouts.DEFAULT_OPERATION_TIMEOUT`

Reply:
No its is not related: it is another client for the tests ;-)